src/ducc0/infra/threading.h

```diff
@@ -240,6 +240,7 @@ template<typename T, typename Func> auto execWorklist
 
 using detail_threading::max_threads;
 using detail_threading::adjust_nthreads;
+using detail_threading::Range;
```

Comment:
Exposing this symbol so we can define a `Scheduler` implementation without needing to refer to anything within the `detail_threading` namespace.

Reply:
OK!